oauth2/handler.go

```diff
@@ -3,12 +3,11 @@ package oauth2
 import (
 	"net/http"
 	"net/url"
-
 	"encoding/json"
-	"strings"
-	"time"
 
 	"github.com/Sirupsen/logrus"
+	"strings"
+	"time"
```

Comment:
I'm not super picky on this but since you probably will take another pass at this anyways, could you run `goimports` on this file? Please on this file only so we get a clean changeset!